hmt/serve/mock/faker/stateless_faker.py

```diff
@@ -170,30 +202,19 @@ def _fake_json(
         )
 
     def _build_full_schema(
-        self, schema: typing.Union[Schema, Reference], spec: OpenAPIObject
+        self, schema: typing.Union[Schema, Reference], definitions: typing.Any
```

Comment:
Are the definitions really `Any`? Wouldn't it be a record of keys to schemas?

Reply:
Yep, it's a dict of dicts.